opam-ci-check/lib/lint.ml

```diff
@@ -309,40 +309,18 @@ module Checks = struct
     get_files dir |> List.map check_file |> List.concat
 
   (** [package_name_collision p0 p1] returns true if [p0] is similar to [p1].
-    Similarity is defined to be either:
+    Similarity is defined to be:
 
     - Case-insensitive string equality considering underscores ([_])
-      and dashes ([-]) to be equal
-    - A Levenshtein distance within 1/6 of the length of the string (rounding up),
-      with names of three characters or less ignored as a special case
-
-    As examples, by this relation:
-
-    - [lru-cache] and [lru_cache] collide
-    - [lru-cache] and [LRU-cache] collide
-    - [lru-cache] and [cache-lru] do not collide
-    - [ocaml] and [pcaml] collide
-    - [ocamlfind] and [ocamlbind] do not collide *)
+      and dashes ([-]) to be equal *)
   let package_name_collision p0 p1 =
     let dash_underscore p0 p1 =
-      let f = function '_' -> '-' | c -> c in
+      let f = function '_' -> '-' | c -> Char.lowercase_ascii c in
```

Comment:
Do we want to tweak this to account for `-/_/<nothing>` as per this [comment](https://github.com/ocurrent/opam-repo-ci/issues/378#issuecomment-2425389474)?

Reply:
Good point. I added that logic in ddde7df and tests for it in 6975fb0 .